cli/src/main/scala/scalafix/cli/Cli.scala

```diff
@@ -79,6 +90,16 @@ case class ScalafixOptions(
     case Some(x) => x
   }
 
+  lazy val resolvedMirror: Option[ScalafixMirror] = (classpath, sourcepath) match {
+    case (Some(cp), Some(sp)) =>
+      Some(ScalafixMirror.fromMirror(scala.meta.Mirror(cp, sp)))
+    case (None, None) => None
+    case _ =>
+      // FIXME: improve during review.
+      throw new Exception(
```

Comment:
Since this requires two options to be set in order to succeed, it's not clear how to do that here.